### PR TITLE
#BE-4784 Add testing groups option

### PR DESCRIPTION
**Pull request by hiilhan**
Depends on #7.

### CODE REVIEW

**Review by burakberkkeskin — DISMISSED** [review]
Tested with a freshly created testing distribution profile.
- The TD profile is created ✅ 
- The "auto send" feature is enabled.
- The tester group(s) are added to the TD profile.
- The TD emails has been received by emails.

**Review by burakberkkeskin — CHANGES_REQUESTED** [review]
Some test cases have been failed. 
Details -> [Linear comment](https://linear.app/appcircle/issue/BE-4784/[fastlane-plugin]provide-an-option-to-input-the-testing-group-name-via#comment-782a8584)

File: README.md [hunk 1]
```diff
@@ -85,7 +85,8 @@ fastlane add_plugin appcircle_testing_distribution
     profileCreationSettings: {
       authType: "${AC_PROFILE_AUTH_TYPE}",
       username: "${AC_PROFILE_USERNAME}",
-      password: "${AC_PROFILE_PASSWORD}"
+      password: "${AC_PROFILE_PASSWORD}",
+      testingGroupNames: "AC_PROFILE_TESTING_GROUPS"
```
Comment (csonuryilmaz) [inline README.md]:
```suggestion
      testingGroupNames: "AC_PROFILE_TESTING_GROUP_NAMES"
```

File: fastlane/Fastfile [hunk 2]
```diff
@@ -9,7 +9,8 @@ lane :test do
     profileCreationSettings: {
       authType: ENV["AC_PROFILE_AUTH_TYPE"],
       username: ENV["AC_PROFILE_USERNAME"],
-      password: ENV["AC_PROFILE_PASSWORD"]
+      password: ENV["AC_PROFILE_PASSWORD"],
+      testingGroupNames: ENV["AC_PROFILE_TESTING_GROUPS"]
```
Comment (csonuryilmaz) [inline fastlane/Fastfile]:
```suggestion
      testingGroupNames: ENV["AC_PROFILE_TESTING_GROUP_NAMES"]
```

File: lib/fastlane/plugin/appcircle_testing_distribution/actions/appcircle_testing_distribution_action.rb [hunk 3]
```diff
@@ -160,12 +161,15 @@ def self.available_options
                                          value[:authType] = value[:authType].to_i
                                          value[:username] ||= ENV["AC_PROFILE_USERNAME"]
                                          value[:password] ||= ENV["AC_PROFILE_PASSWORD"]
+                                         value[:testingGroupNames] ||= ENV["AC_PROFILE_TESTING_GROUPS"]
```
Comment (csonuryilmaz) [inline lib/fastlane/plugin/appcircle_testing_distribution/actions/appcircle_testing_distribution_action.rb]:
```suggestion
                                         value[:testingGroupNames] ||= ENV["AC_PROFILE_TESTING_GROUP_NAMES"]
```

**Review by csonuryilmaz — CHANGES_REQUESTED** [review]

**Pull request merged**